_posts/2015-12-17-community-talks.md

```diff
@@ -17,11 +17,11 @@ As 2015 comes to a close, I'd love to give a big shout-out to the members of the
 * [Jake Weisz](https://twitter.com/ocdtrekkie), who [packaged many apps](https://apps.sandstorm.io/author/Jacob%2520Weisz) and is super helpful in answering community questions on [sandstorm-dev](https://groups.google.com/forum/#!forum/sandstorm-dev), spoke at the [Chicago GNU/Linux Users Group](https://chicagolug.org/tag/jake-weisz.html) as well as [Open Source Open Mic](http://www.meetup.com/Chicago-Open-Source-Open-Mic/events/224632294/).
 * [Deb Nicholson](http://mediagoblin.org/news/deb-nicholson-oscon-award.html) and [Chris Webber](http://dustycloud.org/) (of [MediaGoblin](https://apps.sandstorm.io/app/70awyqss6jq2gkz7dwzsnvumzr07256pzdt3hda9acfuxwh6uh7h)) gave a Sandstorm demo during [their FOSDEM talk](http://ftp.osuosl.org/pub/fosdem//2015/devroom-distributions/can_distros_make_the_link%3f_lets_package_the_customizable,_free_software_web_of_the_future!_.mp4) (Sandstorm demo starts at 24:57).
 
-I made this list based on [the page on the wiki](https://github.com/sandstorm-io/sandstorm/wiki/Sandstorm-Talks). If you also gave a talk about Sandstorm this year and are just now adding yourself to the wiki, drop me a line at [community@sandstorm.io](mailto:community@sandstorm.io), and I'll update the post to include you. A huge thanks to everyone who has given a talk, as well as the folks in the brigade who show up everywhere on Hacker News, reddit, and request coverage of Sandstorm on the [Linux Action Show](http://www.jupiterbroadcasting.com/90881/thankfully-its-open-source-las-393/). I heart all of you. If any of you happen to live in or travel through the SF Bay Area, you should share your work at a [Sandstorm meetup](http://www.meetup.com/Sandstorm-SF-Bay-Area/) or at least drop by.
+I made this list based on [the page on the wiki](https://github.com/sandstorm-io/sandstorm/wiki/Sandstorm-Talks). If you also gave a talk about Sandstorm this year and are just now adding yourself to the wiki, drop me a line at [the mailing list](https://groups.io/g/sandstorm-user-group), and I'll update the post to include you. A huge thanks to everyone who has given a talk, as well as the folks in the brigade who show up everywhere on Hacker News, reddit, and request coverage of Sandstorm on the [Linux Action Show](http://www.jupiterbroadcasting.com/90881/thankfully-its-open-source-las-393/). I heart all of you. If any of you happen to live in or travel through the SF Bay Area, you should share your work at a [Sandstorm meetup](http://www.meetup.com/Sandstorm-SF-Bay-Area/) or at least drop by.
```

Comment:
nit: "on the mailing list" I think

Reply:
Made some adjustments in the next commit. I probably also introduced *more* inconsistency, but for blog posts I am not too worried.